content/Overview/get-started.mdx

```diff
@@ -0,0 +1,57 @@
+---
+id: get-started
+slug: get-started
+title: Get Started
+---
+
+import CliInstallSnippet from '../components/cli-install.mdx';
+
+Learn how to quickly get started with Configu.
+We'll use the Configu's CLI and the Configu's Saas Configuration Store to decalre, initalize, and export a simple set configuration data.
+
```

Comment:
It might be useful to add a sentence about the modular capability, so that new users can onboard/start working with Configu as they need it, not having to change the entire configuration at once.